pkg/tests/testinfra/testinfra.go

```diff
@@ -157,6 +157,14 @@ func CreateGrafDir(t *testing.T, opts ...GrafanaOpts) (string, string) {
 	viewsDir := filepath.Join(publicDir, "views")
 	err = fs.CopyRecursive(filepath.Join(rootDir, "public", "views"), viewsDir)
 	require.NoError(t, err)
+	// Copy index template to index.html, since Grafana will try to use the latter
+	err = fs.CopyFile(filepath.Join(rootDir, "public", "views", "index.html"),
```

Comment:
@ryantxu is this line ever needed now?

Reply:
it *should* not be -- but looks like some tests fail without it, but i think that is a problem with the tests/setup.  I'll look more soon